src/game/engine/overlay.cpp

```diff
@@ -0,0 +1,196 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Chronoshift is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "overlay.h"
+#include "cell.h"
+#include "iomap.h"
+#include "globals.h"
+
+#ifndef GAME_DLL
+TFixedIHeapClass<OverlayClass> g_Overlays;
+#endif
+
+HousesType OverlayClass::s_ToOwn = HOUSES_NONE;
+
+/**
```

Comment:
Don't bother with comment blocks for ctors and operators, its clear what they do and they don't need commenting for the most part.